wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
+	}
+	table#toc tr[style="background:#ccccff"] {
+		background-color: var(--purple-1) !important;
+	}
+	table[class="{{{2}}}"][style="border:1px dotted #4682B4; padding:5px; background-color:#F0F8FF"], table[class="{{{2}}}"][style="border:1px dotted #4682B4;padding:5px;background-color: #F0F8FF;"] {
+		background-color: var(--blue-3) !important;
+		border-color: var(--blue-4) !important;
+	}
+	table[class="{{{2}}}"][style="border:1px dotted #B48246; padding:5px; background-color:#FCC"], table[class="{{{2}}}"][style="border:1px dotted #B48246;padding:5px;background-color: #FCC;"] {
+		background-color: var(--red-0) !important;
+	}
+	.suggestion-thumbnail {
+		background-color: var(--blue-2);
+	}
+	table.mbox-small.plainlinks.sistersitebox {
+		background-color: var(--gray-2) !important;
+		color: var(--white) !important;
+	}
+	.mw-parser-output .spoken-wikipedia, .mw-parser-output .sidebar {
+		background-color: var(--gray-2);
+		border-color: var(--gray-69);
+	}
+	[src="//upload.wikimedia.org/wikipedia/commons/thumb/4/47/Sound-icon.svg/45px-Sound-icon.svg.png"] {
+		filter: invert(.835);
+	}
+    th.sidebar-heading[style="background: #aaddff;color:#000;"], th.sidebar-heading[style="background:#aaddff; color:#000;"] {
+        background-color: var(--blue-4) !important;
+		color: var(--white) !important;
+    }
+	.mw-parser-output .wikipedia-languages-prettybars {
+		background-color: var(--blue-4);
+	}
+	.ws-noexport.noprint[style="float: right; border: 1px solid skyblue; padding-left:0.5em; padding-right:0.5em; background-color: #F3F3F3; margin: 0 0 0.5em 0.5em;"] {
+		background-color: var(--gray-2) !important;
+		border-color: var(--blue-4) !important;
+	}
+	[style="clear:both; background-color:#F7FAFC; margin: 0 auto; text-align:center;"] {
+		background-color: var(--blue-2) !important;
+	}
+	th.infobox-above[style="background-color: #CEDEFF;"] {
+		background-color: var(--blue-4) !important;
+	}
+	[src="//upload.wikimedia.org/wikipedia/commons/thumb/6/66/IPv4_address_structure_and_writing_systems-en.svg/300px-IPv4_address_structure_and_writing_systems-en.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/1/15/Ipv6_address.svg/300px-Ipv6_address.svg.png"] {
+		filter: invert(1);
+		background-color: transparent;
+	}
+	span.citation-needed-content[style="padding-left:0.1em; padding-right:0.1em; color:#595959; border:1px solid #DDD;"] {
+		color: var(--gray-a) !important;
+		border-color: var(--gray-69) !important;
+	}
+	[src="//upload.wikimedia.org/wikipedia/commons/thumb/6/6f/Octicons-terminal.svg/24px-Octicons-terminal.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/8/88/At_sign.svg/15px-At_sign.svg.png"], [src="/static/images/mobile/copyright/wikipedia-wordmark-fr.svg"], [src="/static/images/mobile/copyright/wikipedia-tagline-pt.svg"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/b/bd/Wikipedia-logo-v2-bw.svg/100px-Wikipedia-logo-v2-bw.svg.png"], .mw-ui-icon-wikimedia-language:before {
```

Comment:
NO [src="//upload.wikimedia.org/wikipedia/commons/thumb/6/6f/Octicons-terminal.svg/24px-Octicons-terminal.svg.png"]

YES [src="Octicons-terminal"]

Reply:
and all the other code in here is just, not up to scratch, much needs improving/condensing and generally study the existing style and try to emulate the way overrides are done.